dbt_project.yml

```diff
@@ -78,6 +78,7 @@ vars:
     snowplow__enable_cwv: false
     snowplow__cwv_days_to_measure: 28
     snowplow__cwv_percentile: 75
+    snowplow__cwv_context: "{{ source('atomic','unstruct_event_com_snowplowanalytics_snowplow_web_vitals_1') }}"
```

Comment:
I think this doesn't work with the get_sde_or_context macro, it needs to just be the raw table name (which is also what I need to do with the base macro part anyway with the rest of them)